man/avahi.service.5.xml.in

```diff
@@ -96,7 +96,37 @@
       </option>
 
       <option>
-        <p><opt>&lt;txt-record&gt;</opt> DNS-SD TXT record data.</p>
+        <p><opt>&lt;name replace-wildcards="yes|no"&gt;</opt> The
+        service name. If <opt>replace-wildcards</opt> is "yes", any
+        occurence of the string "%h" will be replaced by the local
+        host name. This can be used for service names like "Remote
+        Terminal on %h". If <opt>replace-wildcards</opt> is not
+        specified, defaults to "no".</p>
+      </option>
+
+      <option>
+        <p><opt>&lt;txt-record
+        value-format="text|binary-hex|binary-base64"&gt;</opt> DNS-SD
+        TXT record data. If <opt>value-format</opt> is "text", the
+        value of the TXT record is taken verbatim. If
+        <opt>value-format</opt> is "binary-hex" then the value of TXT
+        record is decoded by taking pairs of characters after the "="
+        char and interpreting them as the textual representation of
+        the two-digit hexadecimal number. Both uppercase and lowercase
+        hexadecimal digits are allowed. The 0x or 0X prefix is not
+        allowed. This requires the length of the value to be even. If
+        <opt>value-format</opt> is "binary-base64" then the value of
+        TXT record is decoded with base64 decoder. The character set
+        used is A-Za-z0-9+/. This requires the length of the value to
+        be a multiple of 4, with "=" as a padding. If
```

Comment:
‘padding’ sounds more natural to me than ‘the padding’. I think it’s because ‘padding’ is uncountable, but your knowledge of English grammar is probably better than mine.